CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Cromwell Change Log
 
+## 66 Release Notes
+
+### Google Artifact Registry Support
+Cromwell now supports call caching when using Docker images hosted on
+[Google Artifact Registry](https://cloud.google.com/artifact-registry). 
+
 ## 65 Release Notes
 
 * An additional set of metrics relating to metadata age were added.
-* No user facing changes in Cromwell 65.  
```

Comment:
If removing this line be sure to [also edit the releases page](https://github.com/broadinstitute/cromwell/releases/tag/65).

Reply:
Done